### PR TITLE
Add option to select styling sets when creating projects

### LINKED ISSUE #147: Allow to select from two ITCSS versions - full and minimal
- full - what we have currently in master
- minimal - just reset, setup & mixins

@thymikee can you please list which files exactly you'd like to keep in minimal ITCSS version?

### PR DESCRIPTION
**Pull request by jakub300**
Hello, changes in generator.js explain things quite well. Please, let me know what do you think.

Please do not merge for now, only discuss.

Fixes #147

### CODE REVIEW

**Comment by luboskmetko** [comment]
@jakub300 haven't tested this yet but is this to solve #147 and #153?

A quick note - I would rename in prompts `Full (and style guide)` to `Full with style guide`

Will test this later today.

**Comment by jakub300** [comment]
Correct, this should resolve both.
I'll rename it.

**Comment by thymikee** [comment]
How about this wording?

screen shot 2017-05-19 at 15 05 07

**Comment by luboskmetko** [comment]
Is there anything complex about ITCSS?  :) I would rather avoid this word. What about something more verbose like this:

```
Which ITCSS starter files you'd like to use:
- Just absolutely minimum
- All of them
- All of them plus create a Style Guide for me
```

**Comment by jakub300** [comment]
@thymikee @luboskmetko ping

**Comment by thymikee** [comment]
I like the idea, it's just the wording that would use slight adjustments. Are you good with what Lubos suggested?

**Comment by jakub300** [comment]
I don't like the term "absolute minimum" because the things we include by default go beyond that. Things like post, footer, header, main navigation styling are included in the first option but imho they are not "absolute minimum".

**Comment by luboskmetko** [comment]
If that's so, I agree that's not "absolute minimum". Currently I don't have time to study what's included in each option so it would be good to make an exact list of that also for documentation purposes. Then we can think of better labels.

**Comment by jakub300** [comment]
Stylelint is in listing even if not added

**Comment by luboskmetko** [comment]
@jakub300 sorry for delay on this. Looks good to me, just a few things:

- [ ] Minimal and Full options still generate link to the Style guide on the project index page. It points to an non-existent page.
- [ ] I would change the order of options to _Full (with style guide), Full_ and _Minimal_ and make _Full (with style guide)_ selected by default. It would be less confusing if we keep default option generating styles like it was before. 
- [ ] Maybe change _How much styling should be included?_ to _How much ITCSS styling should be included?_. Not sure if it makes clearer, though.

**Comment by jakub300** [comment]
Updated as requested, pushed as separated commit so you can see differences.
Differences in test are mostly due to renames: `defaults.test.js -> styling.minimal.test.js` and `styling.full_styleguide.test.js -> defaults.test.js` which git catched differently.
